solr/core/src/java/org/apache/solr/cloud/overseer/CollectionMutator.java

```diff
@@ -99,7 +100,7 @@ public ZkWriteCommand deleteShard(final ClusterState clusterState, ZkNodeProps m
   }
 
   public ZkWriteCommand modifyCollection(final ClusterState clusterState, ZkNodeProps message) {
-    if (!checkCollectionKeyExistence(message)) return ZkStateWriter.NO_OP;
+    if (!checkCollectionKeyExistence(message) || !checkKeyExistence(message, ZkStateReader.COLLECTION_CONFIG_PROP)) return ZkStateWriter.NO_OP;
```

Comment:
This change is curious to me.    Does this mean the configSet (aka configName in much of the naming) is mandatory for modifying a collection?  If so, that seems wrong.